Gemfile

```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 gemspec
 
 group :test do
-  gem 'rails', '~> 4.0'
+  gem 'rails', '>= 4.0', '< 7.0'
```

Comment:
Since we need to use Rails 7 don't we need to test this using Rails 7? https://newrope.slack.com/archives/C02S08WDQ/p1681097711254339?thread_ts=1680764615.995429&cid=C02S08WDQ

Reply:
Yes, I thought about it last night before bed. I wanted to make sure we don't use Rails 7 yet for debugging this old dependency but actually might not be a wise decision. I will remove the lock.